workflow/templates/cnv_html_report/style.css

```
@@ -235,8 +235,22 @@ svg .gridline {
 svg .gridline.baseline {
-  stroke: #777777;
+  stroke: #aaaaaa;
+  stroke-width: 1px;
+  opacity: 0.5;
+}
+
+svg .integer-cn-gridline {
+  stroke: #4a90d9;
+  stroke-width: 1px;
+  stroke-dasharray: 4, 3;
+  shape-rendering: crispEdges;
```

Stylelint (17.14.0)

[error] Expected "crispEdges" to be "crispedges" (value-keyword-case)

(value-keyword-case)